app/templates/gulpfile.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+// Generated on <%= (new Date).toISOString().split('T')[0] %> using <%= pkg.name %> <%= pkg.version %>
+
+var gulp = require('gulp'),
+    $ = require('gulp-load-plugins')(),
+    browserSync = require('browser-sync'),
+    runSequence = require('run-sequence'),
+    del = require('del'),
+    neat = require('node-neat').includePaths;
```

Comment:
This style of variable assignment is different than in `index.js`.

This is pretty nitpicky, but I think we should have consistency in style. FWIW, I vote for the other style.